### ISSUE TITLE
Migrate the toolbox to siibra-api v3.

### ISSUE THREAD

**Issue by dickscheid** [FZJ-INM1-BDA/siibralab#43]
As a next step I suggest to migrate the toolbox to the latest siibra-api version and do some clean-ups to have the API calls at a few well documented places in the code. Most of other issues will become much easier afterwards.

**Comment by dickscheid** [comment]
@xgui3783 do you agree that migrating to v3 makes sense as a next touch-up?

**Comment by xgui3783** [comment]
I agree. regarding migrating from v1/v2 endpoint to v3 endpoint.

**Comment by xgui3783** [comment]
Some additional information

- swagger UI for siibra-api v3 is at https://siibra-api-stable.apps.hbp.eu/v3_0/docs 
- siibra-explorer currently uses v3 endpoint

**Comment by vijayiyer05** [comment]
@dickscheid can you clarify if your note ...

> The feature request to neurotransmitter receptor densities should then be structured in the style of a template, in order to add other feature modalities of "tabular" type (cell densities, cortical profiles, connectivity matrices, ...) afterwards with minimal effort.

... refers to #35? Or something else or new? My inclination is to delineate what's the over-arching goal (move to API v3) and specific  updates/validations associated to that (such as receptor densities).

**Comment by dickscheid** [comment]
> @dickscheid can you clarify if your note ...
> 
> > The feature request to neurotransmitter receptor densities should then be structured in the style of a template, in order to add other feature modalities of "tabular" type (cell densities, cortical profiles, connectivity matrices, ...) afterwards with minimal effort.
> 
> ... refers to #35? Or something else or new? My inclination is to delineate what's the over-arching goal (move to API v3) and specific updates/validations associated to that (such as receptor densities).

Yes it refers to #35. API v3 will return those as tabular data (a pandas DataFrame on the python side), and for future updates we will have a couple of modalities matching the same principle (same API call, similar tabular data item, differing mostly by modality and a few other metadata fields).

**Comment by vijayiyer05** [comment]
Commenting here on the Issue to say: good progress on #46 by @valentin-pinkau ! Some notes there re some remaining warnings/errors/puzzles before I'd be inclined to declare victory.

Besides the refactor to use API v3, I wonder @dickscheid if there's any small update you/team might propose that would harness v3 in some way. I see cell densities was an original idea 👆🏼. I recognize there's limited hours left, so if that's nontrivial, something else might be suggested. Even if's a minor/token thing, it's nice to have some advance along with a refactor type update.
